jenkins.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: jenkins
+  version: 2.378
+  epoch: 0
+  description:
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation:
+      license: MIT
+  dependencies:
+    runtime:
+      - ttf-dejavu
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+      - coreutils
+      - bash
+      - openssh-client
+      - tini
+      - openjdk-17
+      - maven
+      - tzdata
+      - glibc-locale-en
+      - git
+      - patch
+      - ttf-dejavu
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/jenkinsci/jenkins/archive/refs/tags/jenkins-${{package.version}}.tar.gz
+      expected-sha256: 45f70641da9d1fdb26a2dee6ea201096a0d7da430d67ab782fef0713924530eb
+  - uses: patch
+    with:
+      patches: ignoreArchiveNotReadableTest.patch
+  - runs: |
+      export JAVA_HOME=/usr/lib/jvm/openjdk
+      export LANG=en_US.UTF-8
+
+      # TODO lets see how long running unit tests in CI takes
+      # mvn install -DskipTests=true
+      mvn install
```

Comment:
let's see how long CI takes when running tests

Reply:
is it possible to run the tests in parallel?

Reply:
I've kicked off a couple of alternatives playing with the maven `-DforkCount`, will know in a couple of hours if that helps.

Reply:
Good: `-DforkCount=2`  finished in 1h03m in my tests compared with 2h20m without the fork.  

I did try `-DforkCount=$(nproc)` but got failures 
```
Exception: java.lang.NullPointerException thrown from the UncaughtExceptionHandler in thread "TCP agent connection handler #1 with /192.168.10.2:56910"
```
and
```
JNLPLauncherTest.testDelegatingComputerLauncher:129->launchJnlpAndVerify:190 Agent failed to go online
```

trying a last build with `-DforkCount=4`